auth/users/users-router.js

```diff
@@ -11,14 +12,22 @@ router.post("/register", (req, res) => {
   let userData = req.body;
   const hash = bcrypt.hashSync(userData.password, 8);
   userData.password = hash;
-  console.log(userData);
-  Users.add(userData)
-    .then(user => {
-      res.status(201).json(user);
-    })
+  Users.findBy( userData.user_name )
+  .then(user => {
+    if(!user) {
+      Users.add(userData)
+      .then(user => {
+          res.status(200).json({
+            message: `Registration successful ${user.user_name}!`,
+            id: user.id
+          });
+      })
+    } else {
+      res.status(400).json({ errorMessage: "Username already in use!" });
```

Comment:
I'll double-check later, but I don't believe I've been getting this 400 as a response back.  Though I was trying to avoid causing this.